src/gui/gui.config

```diff
@@ -115,6 +115,19 @@
   </ignition-gui>
 </plugin>
 
+<plugin filename="BananaForScale" name="BananaForScale">
+  <ignition-gui>
+    <property key="resizable" type="bool">false</property>
+    <property key="x" type="double">400</property>
+    <property key="y" type="double">0</property>
+    <property key="width" type="double">100</property>
+    <property key="height" type="double">50</property>
+    <property key="state" type="string">floating</property>
+    <property key="showTitleBar" type="bool">false</property>
+    <property key="cardBackground" type="string">#666666</property>
+  </ignition-gui>
+</plugin>
```

Comment:
As awesome as this plugin is, how about we leave it out of the default GUI configuration? Leave it as an Easter-egg that users can insert from the menu.